### PR TITLE
fix/do not translate nullish value

### PR DESCRIPTION
**Pull request by nicole-eb**

### CODE REVIEW

**Review by Vin-RM — DISMISSED** [review]

File: packages/renderers/lib/src/attribute/widgets/delivery_box_address_attribute_renderer.dart [hunk 1]
```diff
@@ -42,8 +42,9 @@ class DeliveryBoxAddressAttributeRenderer extends StatelessWidget {
                   TranslatedText(value.city, style: valueTextStyle),
                 ],
               ),
-              TranslatedText(valueHints.propertyHints!['country']!.getTranslation(value.country), style: valueTextStyle),
-              TranslatedText(valueHints.propertyHints!['state']!.getTranslation(value.state), style: valueTextStyle),
+              TranslatedText(valueHints.propertyHints!['country']!.getTranslation(value.country)!, style: valueTextStyle),
+              if (valueHints.propertyHints!['state']!.getTranslation(value.state) != null)
+                TranslatedText(valueHints.propertyHints!['state']!.getTranslation(value.state)!, style: valueTextStyle),
```
Comment (jkoenig134) [inline packages/renderers/lib/src/attribute/widgets/delivery_box_address_attribute_renderer.dart:47]: making sth. nullable and then throwing around with unsafe non-null assertions...

**Comment by jkoenig134** [comment]
still not fixing the main issue here. just make sure to not pass `null` to the translate function.

File: packages/renderers/lib/src/attribute/widgets/delivery_box_address_attribute_renderer.dart [hunk 2]
```diff
@@ -42,10 +42,10 @@ class DeliveryBoxAddressAttributeRenderer extends StatelessWidget {
...
                 ],
               ),
-              if (valueHints.propertyHints!['country']!.getTranslation(value.country) != null)
-                TranslatedText(valueHints.propertyHints!['country']!.getTranslation(value.country)!, style: valueTextStyle),
-              if (valueHints.propertyHints!['state']!.getTranslation(value.state) != null)
-                TranslatedText(valueHints.propertyHints!['state']!.getTranslation(value.state)!, style: valueTextStyle),
+              if (valueHints.propertyHints!['country'] != null)
+                TranslatedText(valueHints.propertyHints!['country']!.getTranslation(value.country), style: valueTextStyle),
+              if (valueHints.propertyHints!['state'] != null)
+                TranslatedText(valueHints.propertyHints!['state']!.getTranslation(value.state), style: valueTextStyle),
```
[the remark this answers is not in the document]
Comment (nicole-eb) [reply packages/renderers/lib/src/attribute/widgets/delivery_box_address_attribute_renderer.dart]: value.country can't be null so i removed that check in the new commit :)

**Review by jkoenig134 — APPROVED** [review]

**Comment by jkoenig134** [comment]
this branch is outdated....

**Pull request merged**